dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -35,10 +35,9 @@ def index
   def workshops_user_enrolled_in
     authorize! :workshops_user_enrolled_in, Pd::Workshop
 
-    enrollments = ::Pd::Enrollment.for_user(current_user).all.
-      reject do |enrollment|
-        future_or_current_teachercon_or_fit?(enrollment.workshop)
-      end
+    enrollments = ::Pd::Enrollment.for_user(current_user).all.reject do |enrollment|
+      enrollment.workshop&.future_or_current_teachercon_or_fit?
```

Comment:
I'm surprised this is needed - workshops delete their enrollments when they are deleted. Do we know what happened here?

Reply:
Agreed. This is surprising. Perhaps an anomaly, from some manual intervention? 

The models in question are enrollment id 1460 (created 2016-06-21), and workshop id 292 (created 2016-06-21, deleted 2017-01-06).